source/exec.tex

```diff
@@ -5672,3 +5687,424 @@
 return as_awaitable(std::forward<Value>(value), static_cast<Promise&>(*this));
 \end{codeblock}
 \end{itemdescr}
+
+\rSec1[exec.par.scheduler]{Parallel scheduler}
+
+\pnum
+\tcode{parallel_scheduler} models \libconcept{scheduler}.
+
+\pnum
+Let \tcode{sch} be an object of type \tcode{parallel_scheduler}, and
+let \tcode{\exposid{BACKEND-OF}(sch)} be \tcode{*ptr},
+where \tcode{sch} is associated with \tcode{ptr}.
+
+\pnum
+The expression \tcode{get_forward_progress_guarantee(sch)} returns
+\tcode{forward_progress_guarantee::paral\-lel}.
+
+\pnum
+Let \tcode{sch2} be an object of type \tcode{parallel_scheduler}.
+Two objects \tcode{sch} and \tcode{sch2} compare equal if and only if
+\tcode{\exposid{BACKEND-OF(sch)}} and
+\tcode{\exposid{BACKEND-OF(sch2)}} refer to the same object.
```

Comment:
```suggestion
\tcode{\exposid{BACKEND-OF}(sch)} and
\tcode{\exposid{BACKEND-OF}(sch2)} refer to the same object.
```
I'm actually surprised that checks don't catch this. I remember getting nagged because I wrote `\tcode{\exposid{a-b}}` once, but maybe the check doesn't fire when there are parentheses inside.